packages/transition-backend/src/services/executableJob/ExecutableJob.ts

```diff
@@ -218,7 +217,7 @@ export class ExecutableJob<TData extends JobDataType> extends Job<TData> {
     }
 
     setCancelled(): boolean {
-        if (this.status === 'pending' || this.status === 'inProgress') {
+        if (this.status === 'pending' || this.status === 'inProgress' || this.status === 'paused') {
```

Comment:
we are getting to a point that we need to draw the state diagram as some ascii art in the top of the file.